src/Servers/Kestrel/README.md

```diff
@@ -1,8 +1,39 @@
-KestrelHttpServer
-=================
+# Kestrel
 
-Kestrel is a cross-platform web server for ASP.NET Core.
+Kestrel is our cross-platform web server that is included and enabled by default in ASP.NET Core.
 
-## File logging for functional test
+## Description
 
-Turn on file logging for Kestrel functional tests by specifying the environment variable ASPNETCORE_TEST_LOG_DIR to the log output directory.
+The following contains a description of the sub-directories.
+
+- `Core`: Contains the main server implementation for Kestrel.
+- `Kestrel`: Contains the public API exposed to use Kestrel.
+- `Transport.Libuv`: Contains the Libuv transport for connection management.
+- `Transport.Quic`: Contains the QUIC transport for connection management.
+- `Transport.Sockets`:Contains the Sockets transport for connection management.
```

Comment:
```suggestion
- `Transport.Sockets`: Contains the Sockets transport for connection management.
```

Reply:
What do you think about turning all the directory names into github links?

```suggestion
- [Transport.Sockets](https://github.com/dotnet/aspnetcore/tree/main/src/Servers/Kestrel/Transport.Sockets/src): Contains the Sockets transport for connection management.
```

Reply:
I'm not against it. Should do relative paths here though.

Reply:
Yeah. Relative paths are probably better in case someone is viewing the README locally.